src/main/java/com/littleBeasts/entities/Beast.java

```diff
@@ -12,39 +11,35 @@
 
 import java.awt.*;
 
-@CollisionInfo(collisionBoxWidth = 14, collisionBoxHeight = 8, collision = true)
-
+@CollisionInfo(collisionBoxWidth = 14, collisionBoxHeight = 30, collision = true)
 public class Beast extends Creature implements IUpdateable {
     private CeEntity littleBeast;
```

Comment:
I wouldn't call it littleBeast. Call it what it is a CeEntity or a ceEntityPointer / reference. Same applies for the get method

Reply:
You're right.